depotcommun/settings/base.py

```diff
@@ -128,5 +127,3 @@
 
 STATIC_URL = '/static/'
 
-django_heroku.settings(locals())
```

Comment:
Should we not also remove the `DATABASES` part in base.py?

Reply:
yes that would make it consistent.
will you make the change or should I?